sched/worker/worker.go

```diff
@@ -5,44 +5,19 @@ import (
 	"github.com/scootdev/scoot/sched"
 )
 
-// Create a WorkerController that talks to node
-type WorkerFactory func(node cluster.Node) WorkerController
+// Create a Worker controller that talks to node
```

Comment:
-0, I kept it there intentionally because it helps clarify the purpose vs code living under /workerapi.